css/style.css

```diff
@@ -0,0 +1,327 @@
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  color: white;
+  background: #3e3c3c;
+  padding: 0;
+}
+
+.site-header > .icon {
+  display: none;
+}
+
+.about-skills > div > h3 {
+  margin: 0 0 0 15px;
+  padding: 0;
+  color: white;
+}
+
+#title-text > h3 {
+  float: left;
+  margin: 0;
+  padding: 100px;
+}
+
+.logo-text {
+  display: inline-block;
+  width: auto;
+  height: auto;
+  color: #fff;
+  text-decoration: none;
+}
+
+#navbar {
+  margin-left: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#navbar > a {
+  display: block;
+  width: 70px;
+  text-decoration: none;
+  height: auto;
+  color: white;
+  margin: 3px;
+}
+
+#home-content {
+  display: flex;
+  width: 100%;
+  height: 100vh;
+  background: #1c1a19;
+}
+
+.left-content {
+  width: 60px;
+  height: 100vh;
+  color: white;
+  margin: 10px 25px 5px 5px;
+  padding: 10px 20px 20px 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.desktop-version-list {
+  list-style-type: none;
+  align-self: center;
+}
+
+.desktop-version-list > li {
+  margin: 2px;
+  padding: 0;
+}
+
+.main-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: start;
+  text-align: left;
+  width: 100%;
+  min-height: 100vh;
+  margin: 5px 10px 15px 10px;
+  padding: 40px 10px 120px 15px;
+  flex: 1 1 100%;
+}
+
+.name-text {
+  color: #f55800;
+  font-family: "Crete Round", sans-serif;
+  text-align: left;
+  font-style: normal;
+  font-weight: 400, regular;
+  font-size: 72px;
+  line-height: 82px;
+  margin: 5px 10px 5px 5px;
+  align-self: start;
+}
+
+.job-title {
+  color: #fff;
+  font-family: "Inter", sans-serif;
+  text-align: left;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 5px 10px 5px 5px;
+}
+
+.job-description {
+  color: #fff;
+  text-align: left;
+  font-family: "Inter", sans-serif;
+  font-weight: 400, regular;
+  align-self: start;
+  font-style: normal;
+  font-size: 20px;
+  line-height: 32px;
+  margin-left: 5px;
+}
+
+#skill-text > ul {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding-left: 5px;
+}
+
+.img-icons {
+  width: 25px;
+  height: 25px;
+  margin: 7px;
+  background: white;
+}
+
+#social-icons {
+  display: none;
+}
+
+.right-content {
+  width: 100%;
+  height: 100vh;
+  transform: rotate(-90deg);
+  -ms-transform: rotate(-180deg);
+  -webkit-transform: rotate(-2deg);
+  margin: 0;
+  background-image: url("../images/Illustration-headerpic.png");
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  background-size: auto 600px;
+}
+
+.mobile-image {
+  display: none;
+}
+
+.btn {
+  color: white;
+  font-size: 1.5rem;
+  border: 0;
+  background: 0;
+}
+
+@media All And (max-width: 500px) {
+  .container {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    height: auto;
+    color: white;
+  }
```

Comment:
I suggest you remove this media query. Remember that mobile-first development is the industry standard so you shouldn't have any media queries until you start working on the portfolio's desktop version.

Reply:
In my opinion, it is not needed as this project was approved earlier as this is my second week , and had to repeat this week due to attendance shortage.

Reply:
I do understand the project was approved with this media query in it, however, it shouldn't have been because of what I explained in my previous comment. Kindly fix it, I don't recommend having pieces of code that don't adhere to the industry standards.